packages/neos-ui-editors/src/EditorEnvelope/index.js

```diff
@@ -54,15 +60,22 @@ export default class EditorEnvelope extends PureComponent {
         if (editorDefinition && editorDefinition.component) {
             const EditorComponent = editorDefinition && editorDefinition.component;
 
+            const restProps = omit(this.props, ['validationErrors', 'highlight']);
+            const {highlight, validationErrors} = this.props;
```

Comment:
`const {highlight, validationErrors, ...restProps} = this .props`
That way you wouldn't need the `omit` function call

Reply:
fixed